packages/nr-metadata/nr_metadata/ui/semantic-ui/js/nr/forms/components/SubjectsField/SubjectsModal.jsx

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import { Button, Form, Grid, Modal, Popup, Icon } from "semantic-ui-react";
+import { Formik } from "formik";
+import * as Yup from "yup";
+import { i18next } from "@translations/i18next";
+import PropTypes from "prop-types";
+import { MultilingualTextInput } from "@js/oarepo_ui";
+import { requiredMessage } from "../../../validationschemas/NRDocumentValidationSchema";
```

Comment:
hard dependency on NR docs, where there shouldn't be any (requiredMessage being very generic, could be extracted even to oarepo_ui)

Reply:
Ok moved to oarepo